### ISSUE TITLE
clearOnCompleted whitespaces

### ISSUE THREAD

**Issue by sayem314** [AndiDittrich/Node.CLI-Progress#40]
When using multibar with mentioned option set to true, nothing happens after all the bars have reached total value. Isn’t it supposed to clear the progress?

**Comment by AndiDittrich** [comment]
did you call  `stop()` or set `stopOnComplete` ? if not it cannot work...the option is bound to the stop() call

**Comment by sayem314** [comment]
That worked. But it's leaving so much space tho!

**Comment by AndiDittrich** [comment]
i will check this soon

**Comment by aikoofujimotoo** [comment]
I also experience the same thing.
Behaviour: https://youtu.be/pg96wcosaKg

And here's my multibar config:
```javascript
const multibars = new cliProgress.MultiBar({
      format: '{task} [{bar}] {percentage}% | Elapsed: {duration}s ' +
          '| ETA: {eta}s | {value}/{total}',
      barCompleteChar: '\u2588',
      barIncompleteChar: '\u2591',
      fps: 25, // I prefer fast update on progress.
      cleanOnComplete: true,
      hideCursor: true,
      stopOnComplete: true,
});
```

**Comment by AndiDittrich** [comment]
everything seems to work as expected:

![Peek 2019-09-28 12-11](https://user-images.githubusercontent.com/1474107/65814960-456e5900-e1e9-11e9-9d37-132df723b884.gif)

**Comment by sayem314** [comment]
![Screenshot from 2019-09-28 16-45-49](https://user-images.githubusercontent.com/14138401/65815291-7d889280-e20f-11e9-88ea-0339804655a2.png)

For me, it does not work this way.

**Comment by AndiDittrich** [comment]
zsh ?

**Comment by AndiDittrich** [comment]
clearOnComplete only works in case stop is called (manually or by stopOnComplete)

**Comment by AndiDittrich** [comment]
and which node version do you use (and also which platform) ?

**Comment by sayem314** [comment]
> zsh ?

No. Bash on hyper terminal.

> clearOnComplete only works in case stop is called (manually or by stopOnComplete)

Yeah I'm aware!

> and which node version do you use (and also which platform) ?

`v10.16.3` and ubuntu 18.04.x

**Comment by AndiDittrich** [comment]
the issue seems to be caused by `this.terminal.cursorRestore();` which restores the last (not the stored) cursor position on ssh/remote connections.

it does work as expected on native terminal emulations

**Comment by AndiDittrich** [comment]
please try the current master branch

**Comment by sayem314** [comment]
Fixed thanks :smile:

**Comment by AndiDittrich** [comment]
great. just released v3.2.0 including the fixed as well as #42
